README.md

```diff
@@ -21,6 +21,25 @@ Frontend at Sentry
 
 The frontend codebase is currently located under `src/sentry/static/sentry/app` in sentry and `static/getsentry` in getsentry. (We intend to align to `static/sentry` in future.)
 
+## Folder & File structure
+
+### File Naming
+
+- Name a file meaningfully, based on what they are or do
+- If not necessary, don't use prefixes or sufixes (ie, dataScrubbingEditModal, dataScrubbingAddModal)
```

Comment:
```suggestion
- Unless necessary, don't use prefixes or suffixes (ie. `dataScrubbingEditModal`, `dataScrubbingAddModal`) instead favor names like `dataScrubbing/editModal`.
```

Including an example of what is good could help here.

Reply:
sure :)